storm-core/dependency-reduced-pom.xml

```diff
@@ -0,0 +1,743 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
1: removed file dependency-reduced-pom.xml
2: added  it to the  .gitignore file